src/legacy/ui/public/agg_types/buckets/_terms_helper.tsx

```diff
@@ -35,19 +32,4 @@ const aggFilter = [
   '!sum_bucket',
 ];
 
-// Returns true if the agg is compatible with the terms bucket
-function isCompatibleAgg(agg: AggConfig) {
-  return !aggFilter.includes(`!${agg.type.name}`);
-}
-
-function safeMakeLabel(agg: AggConfig) {
-  try {
-    return agg.makeLabel();
-  } catch (e) {
-    return i18n.translate('common.ui.aggTypes.buckets.terms.aggNotValidLabel', {
-      defaultMessage: '- agg not valid -',
-    });
-  }
-}
-
-export { aggFilter, isCompatibleAgg, safeMakeLabel };
+export { aggFilter };
```

Comment:
There is no necessity to store it in a separate file now. Please, move it to `order_agg.tsx`